app/views/general_shopping_list/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2>Shopping List</h2>
+<p>Amount of food items to buy: <span><%= @foods.length %></span> Total value of food needed: <span>$<%= @total_price_all %></span></p>

```

Comment:
this is the piace of code updated in the branch and it is here

Reply:
Please next review download the branch and be sure to have everything updated is not fair that I'm getting rejected for this problem